topics/appendix/tools/ide_tooling/internal_actions/internal_ui_inspector.md

```diff
@@ -82,3 +82,12 @@ Some additional properties are available when inspecting the <control>Settings</
 | <p><ui-path>Editor &#124; Inlays</ui-path></p><p>[](inlay_hints.md)</p>                                                          | <p><control>Inlay Group Key</control> - [`InlayGroupSettingProvider.getGroup().key`](%gh-ic%/platform/lang-api/src/com/intellij/codeInsight/hints/settings/InlayGroupSettingProvider.kt)</p><p><control>Inlay Group Key</control> - [`InlayGroup.key`](%gh-ic%/platform/lang-api/src/com/intellij/codeInsight/hints/InlayHintsProvider.kt)</p><p><control>Inlay Provider Model ID</control> - [`InlayProviderSettingsModel.id`](%gh-ic%/platform/lang-api/src/com/intellij/codeInsight/hints/settings/InlayProviderSettingsModel.kt)</p><p><control>Inlay ImmediateConfigurable ID</control> - [`ImmediateConfigurable.Case.id`](%gh-ic%/platform/lang-api/src/com/intellij/codeInsight/hints/InlayHintsProvider.kt)</p> |
 | <p><ui-path>Editor &#124; Intentions</ui-path></p><p>[](code_intentions.md)</p>                                                  | <p><control>Intention Class</control> - [`IntentionAction`](%gh-ic%/platform/analysis-api/src/com/intellij/codeInsight/intention/IntentionAction.java) class</p><p><control>Intention description directory</control> - `<descriptionDirectoryName>`</p>                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
 | <p><ui-path>Plugins</ui-path></p><p>[](plugin_configuration_file.md)</p>                                                         | <p><control>Plugin ID</control> - Plugin `<id>`</p><p><control>Plugin Dependencies</control> - IDs of dependent plugins</p>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+
+## Accessibility Checks
+
+The _UI Inspector_ includes a <control>Show Accessibility Issues</control> (2025.2+) button that enables inspections for common accessibility problems, similar to browser tools like Google Chrome Lighthouse or apps like Accessibility Insights for Windows.
```

Comment:
```suggestion
Since 2025.2, the _UI Inspector_ includes the <control>Show Accessibility Issues</control> button that enables inspections for common accessibility problems, similar to browser tools like Google Chrome Lighthouse or apps like Accessibility Insights for Windows.
```